core/src/nodes/handled_node_tasks.rs

```diff
@@ -58,7 +58,7 @@ use void::Void;
 
 /// Implementation of `Stream` that handles a collection of nodes.
 pub struct HandledNodesTasks<TInEvent, TOutEvent, THandler> {
-    /// For each active task, a sender allowing to transmit messages. Closing the sender interrupts
+    /// An FNV hashmap for each active task to a sender allowing to transmit messages. Closing the sender interrupts
```

Comment:
Is the implementation – FNV hashmap – really relevant here? 
```suggestion
    /// A map between active tasks to an unbounded sender, used to control the task. Closing the sender interrupts
```